### PR TITLE
remove 'foo' and other print msg from test

### PR DESCRIPTION
**Pull request by apeforest**
## Description ##
Many messages are printed from test_profiler unit test and clogged CI output. Removing all prints from test.

### CODE REVIEW

File: tests/python/unittest/test_profiler.py [hunk 1]
```diff
@@ -208,9 +210,11 @@ def doLog(counter):
     counter.set_value(5)
     counter += 1
     start = time.time()
-    doLog(counter)
+    log = get_log(counter)
+    assert len(log) == 100000
```
Comment (access2rohit) [inline tests/python/unittest/test_profiler.py]: `assert len(get_log(counter)) == 100000`

File: tests/python/unittest/test_profiler.py [hunk 2]
```diff
@@ -126,23 +123,25 @@ def makeParams():
...
-    logging.basicConfig()
     enable_profiler('test_profile_frame.json')
     python_domain = profiler.Domain('PythonDomain::test_profile_frame')
     frame = profiler.Frame(python_domain, "test_profile_frame")
     frame.start()
     start = time.time()
     var = mx.nd.ones((1000, 500))
-    doLog()
+    log = get_log()
+    assert len(log) == 100000
```
Comment (access2rohit) [inline tests/python/unittest/test_profiler.py]: `assert len(get_log()) == 100000`

File: tests/python/unittest/test_profiler.py [hunk 3]
```diff
@@ -71,52 +69,51 @@ def test_profiler():
...
-    logging.basicConfig()
     enable_profiler('test_profile_task.json')
     python_domain = profiler.Domain('PythonDomain::test_profile_task')
     task = profiler.Task(python_domain, "test_profile_task")
     task.start()
     start = time.time()
     var = mx.nd.ones((1000, 500))
-    doLog()
+    log = get_log()
+    assert len(log) == 10
```
Comment (access2rohit) [inline tests/python/unittest/test_profiler.py]: `assert len(get_log()) == 100000`

File: tests/python/unittest/test_profiler.py [hunk 4]
```diff
@@ -152,23 +151,24 @@ def makeParams():
...
     event = profiler.Event("test_profile_event")
     event.start()
     start = time.time()
     var = mx.nd.ones((1000, 500))
-    doLog()
+    assert len(get_log()) == 100000
     var.asnumpy()
     stop = time.time()
     event.stop()
-    print('run took: %.3f' % (stop - start))
```
Comment (access2rohit) [inline tests/python/unittest/test_profiler.py]: if this is printed only once do we need to remove it ?

Comment (apeforest) [reply tests/python/unittest/test_profiler.py]: yes, there should be no print in unit test

**Comment by karan6181** [comment]
LGTM. Thanks for your contribution!

**Comment by Vikas-kum** [comment]
@apeforest This PR is causing tests to fail . 
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwindows-cpu/detail/master/1030/pipeline/ 

======================================================================

FAIL: test_profiler.test_profile_task

----------------------------------------------------------------------

Traceback (most recent call last):

  File "C:\Python37\lib\site-packages\nose\case.py", line 198, in runTest

    self.test(*self.arg)

  File "C:\jenkins_slave\workspace\ut-python-cpu@2\tests\python\unittest\test_profiler.py", line 115, in test_profile_task

    assert stop > start

AssertionError


======================================================================

FAIL: test_profiler.test_profile_tune_pause_resume

----------------------------------------------------------------------

Traceback (most recent call last):

  File "C:\Python37\lib\site-packages\nose\case.py", line 198, in runTest

    self.test(*self.arg)

  File "C:\jenkins_slave\workspace\ut-python-cpu@2\tests\python\unittest\test_profiler.py", line 178, in test_profile_tune_pause_resume

    test_profile_task()

  File "C:\jenkins_slave\workspace\ut-python-cpu@2\tests\python\unittest\test_profiler.py", line 115, in test_profile_task

    assert stop > start

AssertionError

**Pull request merged**